bindings/nodejs/examples/client/08-data-block.ts

```diff
@@ -39,10 +40,10 @@ async function run() {
         const fetchedBlock = await client.getBlock(blockIdAndBlock[0]);
         console.log('Block data: ', fetchedBlock);
 
-        if (fetchedBlock instanceof BasicBlock) {
+        if (fetchedBlock.type === BlockType.Basic) {
             const basic = fetchedBlock as BasicBlock;
-            if (basic instanceof TaggedDataPayload) {
-                const payload = basic as TaggedDataPayload;
+            if (basic.block.payload instanceof TaggedDataPayload) {
```

Comment:
maybe we should do the same here with the type field

Reply:
Its tricky, since the BasicBlock is a type, i cannot do `instanceof` on it (types are erased at runtime)
But maybe it would be good to align. just dont know what is the standard

Reply:
instanceof seems like it should be bad practice to me

Reply:
im always scared of == in js, as they also have the === :D instanceof is the safe choice haha

Reply:
It's not the safe choice if we're using aliases